genesyscloud/architect_flow/resource_genesyscloud_architect_flow_schema.go

```diff
@@ -49,6 +49,18 @@ func ResourceArchitectFlow() *schema.Resource {
 		},
 		SchemaVersion: 1,
 		Schema: map[string]*schema.Schema{
+			"name": {
+				Description: "Flow name.",
+				Type:        schema.TypeString,
+				Optional:    false,
+				Computed:    true,
+			},
+			"type": {
+				Description: "Flow type. Valid options: " + strings.Join(validFlowTypes, ", "),
```

Comment:
though its optional, you can still add validatorfunc similar to datasource schema of genesyscloud_flow.